versions.md

```diff
@@ -1,6 +1,11 @@
 Forthcoming
 -----------
 
+Version 2.10.0
+-------------
+
+* Implement Discovery Server example
+
```

Comment:
Normally we do not include examples in the versions.md. I think you can just mark that check as *N/A*

```suggestion
```